scripts/copy_clinvar.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# this should be run with high-core low memory resources
+# e.g.
+# analysis-runner analysis-runner --cpu 8 --memory 2Gi scripts/copy_clinvar.sh 2022-09-17
+
+set -ex
+
+DATE=${1}
+CPG_ANNO=${2:-"gs://cpg-reference/seqr/v0-1/"}
```

Comment:
Let's maybe drop this `v0-1` version tag, since we are using the Broad's datestamp versions?